dao/tx/mempool_tx.go

```diff
@@ -132,21 +91,21 @@ func (m *defaultMempoolModel) GetMempoolTxsTotalCount() (count int64, err error)
 	return count, nil
 }
 
-func (m *defaultMempoolModel) GetMempoolTxByTxHash(hash string) (mempoolTx *MempoolTx, err error) {
-	dbTx := m.DB.Table(m.table).Where("status = ? and tx_hash = ?", PendingTxStatus, hash).Find(&mempoolTx)
+func (m *defaultMempoolModel) GetMempoolTxByTxHash(hash string) (tx *Tx, err error) {
+	dbTx := m.DB.Table(m.table).Where("tx_hash = ?", hash).Find(&tx)
```

Comment:
Do we need to apply the `StatusPending` filter here?

Reply:
Better not, because in current design, the tx that pending and executed but didn't packed into the block will be kept in the mempool, so once the tx has been executed, users can get the executed tx by txhash to query the related result.

Reply:
However, he can also find the tx in mempopl. It is kind of wired.